lib/phoenix/channel/server.ex

```diff
@@ -147,9 +147,9 @@ defmodule Phoenix.Channel.Server do
   def reply(pid, ref, topic, {status, payload}, serializer)
       when is_binary(topic) and is_map(payload) do
 
-    send pid, serializer.encode!(
+    send pid, {:channel_push, serializer.encode!(
```

Comment:
My question is why we don't simply send it without the wrapping, since the wrapper is removed anyway. But I think it probably makes sense to be explicit.